01-js-intro-and-jquery/spotify.js

```diff
@@ -0,0 +1,31 @@
+$(document).ready(function(){
+	
+	$("input").focus();
+
+	var searchList = function(query){
```

Comment:
Try to write the functions definitions as

```
var sarasaFunction = function (...) { 
  ... 
};
```

Just for conventions and good reading of the code